schemas/Message.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "Message",
+  "description": "a ledger message",
```

Comment:
This is specifically a `five-bells-ledger` message, right? This is different from the ledger plugin interface message type